src/java/org/apache/cassandra/locator/SimpleStrategy.java

```diff
@@ -100,12 +102,13 @@ public void validateOptions() throws ConfigurationException
     }
 
     @Override
-    public void maybeWarnOnOptions()
+    public void maybeWarnOnOptions(ClientState state)
     {
         if (!SchemaConstants.isSystemKeyspace(keyspaceName))
         {
             int nodeCount = StorageService.instance.getHostIdToEndpoint().size();
             // nodeCount==0 on many tests
+            Guardrails.minimumRF.guard(rf.fullReplicas, keyspaceName, false, state);
```

Comment:
your error messages says datacenter but you are providing a keyspace, can you fix?